samples/servicecontrol/adapter-rabbitmq-different-topologies/Adapter/Program.cs

```diff
@@ -15,7 +15,15 @@ static void Main()
     {
         #region AdapterTransport
 
-        var transportAdapterConfig = new TransportAdapterConfig<RabbitMQTransport, MsmqTransport>("ServiceControl.RabbitMQ.Adapter");
+        var transportAdapterConfig = new TransportAdapterConfig<RabbitMQTransport, RabbitMQTransport>("ServiceControl.RabbitMQ.Adapter");
+
+        transportAdapterConfig.EndpointSideErrorQueue = "adapter_error";
+        transportAdapterConfig.EndpointSideAuditQueue = "adapter_audit";
+        transportAdapterConfig.EndpointSideControlQueue = "adapter_Particular.ServiceControl";
+
+        transportAdapterConfig.ServiceControlSideErrorQueue = "error";
+        transportAdapterConfig.ServiceControlSideAuditQueue = "audit";
+        transportAdapterConfig.ServiceControlSideControlQueue = "Particular.ServiceControl";
```

Comment:
this scares me a little bit, it introduces some queue names complexity just for the sake of demoing how to support `DirectRoutingTopology`. Is there something better we can do?